### ISSUE TITLE
NPE when right-clicking in Horta 3D viewer

### ISSUE THREAD

**Issue by carshadi** [JaneliaSciComp/hortacloud#20]
Occasionally, when first loading a workspace, I get this error when right-clicking in the 3D viewer to bring up the rendering options (not on a neuron) 
```
java.lang.NullPointerException
	at org.janelia.horta.NeuronTracerTopComponent$13.createMenu(NeuronTracerTopComponent.java:1571)
	at org.janelia.horta.NeuronTracerTopComponent$13.showPopup(NeuronTracerTopComponent.java:1804)
	at org.openide.awt.MouseUtils$PopupMouseAdapter.maybePopup(MouseUtils.java:189)
	at org.openide.awt.MouseUtils$PopupMouseAdapter.mouseReleased(MouseUtils.java:184)
	at java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:290)
	at java.awt.Component.processMouseEvent(Component.java:6539)
	at java.awt.Component.processEvent(Component.java:6304)
	at java.awt.Component.dispatchEventImpl(Component.java:4889)
	at java.awt.Component.dispatchEvent(Component.java:4711)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:760)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:84)
	at java.awt.EventQueue$4.run(EventQueue.java:733)
	at java.awt.EventQueue$4.run(EventQueue.java:731)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:730)
[catch] at org.janelia.workstation.common.logging.EDTExceptionInterceptor.dispatchEvent(EDTExceptionInterceptor.java:21)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)

```

This is resolved by selecting an arbitrary point on a neuron, then right-clicking again away from the neuron to bring up the rendering options

**Comment by olbris** [comment]
This is an issue we know about, but unfortunately, we've never been able to reproduce it on command and get any traction addressing it.

**Comment by olbris** [comment]
Fixed for 9.14.
